### ISSUE TITLE
Tesstrain.sh Unrecognized argument 

### ISSUE THREAD

**Issue by mletni** [tesseract-ocr/tesseract#1758]
1. Environment

**Tesseract**
Tesseract Version = 4.0.0-beta.3
Commit Number = a078ce0 (full: a078ce02bb97a303e396e3863b3e2878deb6c5db)
installation-command path:
![leptonicainstallation](https://user-images.githubusercontent.com/34307768/42377530-f486916e-8123-11e8-9747-5d28593ca337.PNG)

**Leptonica**
Leptonica Version = 1.76
Commit Number = 4cfca60 (full:4cfca60a8aaec1bb1ad3a769005be12a68ad1709)
installation-command path:
![tesseractinstallation](https://user-images.githubusercontent.com/34307768/42377538-f83df662-8123-11e8-9ed5-3660e9568949.PNG)

**Libraries**
main libraries that are installed for sure:
- autoconf
- automake libtool 
- autoconf-archive 
- pkg-config
- libpng12-dev 
- libjpeg8-dev 
- libtiff5-dev 
- zlibig-dev
- libicu-dev 
- libpango1.0-dev 
- libcairo2-dev 
- libleptonica-dev 
- xzgv
- g++ 
- clang++ 

Each library was updated on 2018.07.06 12:30(UTC+01:00) [07:55(GMT-4(Washington))]

**Platform**
Platform = Linux Mint 18.3 Sylvia x64b

2.Current Behavior:

Throws the error:
ERROR: Unrecognized argument --linedata_only 

3.Expected Behavior:

You wrote: "Using tesstrain. The setup for running tesstrain.sh is the same as for base Tesseract. 
Use --linedata_only option for LSTM training."

quotes from: "https://github.com/tesseract-ocr/tesseract/wiki/TrainingTesseract-4.00"

4.Suggested Fix:

Maybe there is a missing variable in tesstrain.sh, or unclear way to use this flag in manual.

**Comment by Shreeshrii** [comment]
You probably have an old version of tesstrain.sh installed in your path.

The files in tesseraact/src/training are the most current ones. I don't know if that is the version being installed.

@AlexanderP Please let us know the location where debian/ubuntu distribution puts tesstrain.sh.

**Comment by AlexanderP** [comment]
@Shreeshrii 
package: libtesseract-dev
path: /usr/share/tesseract-ocr/tesstrain.sh

https://notesalexp.org/packages/en/stretch/amd64/libtesseract-dev/filelist.html

**Comment by Shreeshrii** [comment]
@AlexanderP Thanks!

@mletni Please check that you have the current version installed and remove any files installed by older versions.

**Comment by mletni** [comment]
@Shreeshrii, @AlexanderP  thank you for your reply.

First, i install libtesseract-dev.

Now flag --linedata_only is working.

BUT 

I have two different tesstrain.sh and tesstrain_utils.sh

There are differences:

/usr/share/tesseract-ocr/tesstrain.sh  VS tesseract/src/training/tesstrain.sh

![tesstrain](https://user-images.githubusercontent.com/34307768/42452995-d53c9000-838b-11e8-889d-96dc5dda02cf.png)

AND

/usr/share/tesseract-ocr/tesstrain_utils.sh  VS tesseract/src/training/tesstrain_utils.sh

![utils 3](https://user-images.githubusercontent.com/34307768/42453046-ef83866c-838b-11e8-9a33-418618e4ff48.PNG)
.
.
.
![utils 2](https://user-images.githubusercontent.com/34307768/42453090-f9af1660-838b-11e8-8499-9fc65202f708.PNG)
.
.
.
![utils 1](https://user-images.githubusercontent.com/34307768/42453098-fdc268c4-838b-11e8-866f-5c5bb6fdf5da.PNG)

ALSO

Now i cannot call tesstrain.sh as function(just, anywhere type tesstrain.sh [options]), i have to go to tesseract/src/training and call ./tesstrain.sh --lang... --fonts... e.t.c

HOWEVER it also doesnt work

It trow error: ERROR: tmp/tmp.0sB4jpLJvL/pol./pol.font_name.exp0.box does not exist or is not readable.
I wannt to say part "/tmp.0sB4jpLJvL/" it's different for every single launch of tesstrain.sh


Thanks for your help! Best regards :)

**Comment by Shreeshrii** [comment]
Yes, the versions will be different because the installed version is from the version that was shipped for ubuntu.

The version is src/ is the version currently under development (not released).

Your earlier error was related to having an old version of tesstrain.sh (for tesseract3)  in /usr/share/tesseract-ocr/

The error you are getting now is related to the  training text and font that you are using.
